doc/tutorials/tarball_installation.rst

```diff
@@ -0,0 +1,249 @@
+
+      sudo blkid /dev/nvme0n1p2
+
+   In the output, look for the UUID and PARTUUID (example below). You will need
+   them in the next step.
+
+   .. code-block:: console
+
+      /dev/nvme0n1p2: UUID="3cac5675-e329-4cal-b346-0a3e65f99016" TYPE="ext4" PARTUUID="03db7f45-8a6c-454b-adf7-30343d82c4f4"
+
+#. Add the ACRN Service VM to the GRUB boot menu:
+
+   a. Edit the GRUB ``40_custom`` file. The following command uses ``vi``, but
+      you can use any text editor.
+
+      .. code-block:: bash
+
+         sudo vi /etc/grub.d/40_custom
+
+   #. Add the following text at the end of the file. Replace ``UUID`` and
+      ``PARTUUID`` with the output from the previous step. Replace ``<board>``
+      and ``<scenario>`` to match your bin file.
+
+      .. code-block:: bash
+         :emphasize-lines: 6,8
+
+         menuentry "ACRN Multiboot Ubuntu Service VM" --id ubuntu-service-vm {
+           load_video
+           insmod gzio
+           insmod part_gpt
+           insmod ext2
+           search --no-floppy --fs-uuid --set "UUID"
+           echo 'loading ACRN...'
+           multiboot2 /boot/acrn/acrn.<board>.<scenario>.bin  root=PARTUUID="PARTUUID"
+           module2 /boot/vmlinuz-5.10.65-acrn-service-vm Linux_bzImage
+         }
+
+      Example:
+
+      .. code-block:: console
+
+         menuentry "ACRN Multiboot Ubuntu Service VM" --id ubuntu-service-vm {
+           load_video
+           insmod gzio
+           insmod part_gpt
+           insmod ext2
+           search --no-floppy --fs-uuid --set "3cac5675-e329-4cal-b346-0a3e65f99016"
+           echo 'loading ACRN...'
+           multiboot2 /boot/acrn/acrn.my_board.shared.bin  root=PARTUUID="03db7f45-8a6c-454b-adf7-30343d82c4f4"
+           module2 /boot/vmlinuz-5.10.65-acrn-service-vm Linux_bzImage
+         }
+
+   #. Save and close the file.
+
+#. Make the GRUB menu visible when
+   booting and make it load the Service VM kernel by default:
+
+   a. Edit the ``grub`` file:
+
+      .. code-block:: bash
+
+         sudo vi /etc/default/grub
+
+   #. Edit lines with these settings (comment out the ``GRUB_TIMEOUT_STYLE``
+      line). Leave other lines as they are:
+
+      .. code-block:: bash
+
+         GRUB_DEFAULT=ubuntu-service-vm
+         #GRUB_TIMEOUT_STYLE=hidden
+         GRUB_TIMEOUT=5
+
+   #. Save and close the file.
+
+#. Update GRUB and reboot the system:
+
+   .. code-block:: bash
+
+      sudo update-grub
+      reboot
+
+#. Confirm that you see the GRUB menu with the "ACRN Multiboot Ubuntu Service
+   VM" entry. Select it and proceed to booting ACRN. (It may be autoselected, in
+   which case it will boot with this option automatically in 5 seconds.)
+
+   .. code-block:: console
+      :emphasize-lines: 8
+
+                                GNU GRUB version 2.04
+      ────────────────────────────────────────────────────────────────────────────────
+      Ubuntu
+      Advanced options for Ubuntu
+      Ubuntu 18.04.05 LTS (18.04) (on /dev/nvme0n1p2)
```

Comment:
Should we change this to show Ubuntu 20.04?

Reply:
> Should we change this to show Ubuntu 20.04?

Yes, we should